localstate/localstate.go

```diff
@@ -127,17 +127,10 @@ func (ls *LocalState) RemoveBuilder(builderName string) error {
 	if err != nil {
 		return err
 	}
-
-	eg, _ := errgroup.WithContext(context.TODO())
 	for _, fi := range fis {
-		func(fi os.DirEntry) {
-			eg.Go(func() error {
-				return ls.RemoveBuilderNode(builderName, fi.Name())
-			})
-		}(fi)
-	}
-	if err := eg.Wait(); err != nil {
-		return err
+		if err := ls.RemoveBuilderNode(builderName, fi.Name()); err != nil {
```

Comment:
Can you add a comment to this that indicates it isn't safe to call this method from multiple goroutines at the same time?